paymentsheet/src/test/java/com/stripe/android/lpmfoundations/paymentmethod/PaymentMethodMetadataTest.kt

```diff
@@ -1236,34 +1279,20 @@ internal class PaymentMethodMetadataTest {
             paymentMethodSaveConsentBehavior = PaymentMethodSaveConsentBehavior.Legacy
         )
 
-        assertThat(
-            metadataForPaymentIntent.allowRedisplay(PaymentSelection.CustomerRequestedSave.RequestReuse)
-        ).isEqualTo(PaymentMethod.AllowRedisplay.UNSPECIFIED)
-
-        assertThat(
-            metadataForPaymentIntent.allowRedisplay(PaymentSelection.CustomerRequestedSave.RequestNoReuse)
-        ).isEqualTo(PaymentMethod.AllowRedisplay.UNSPECIFIED)
-
-        assertThat(
-            metadataForPaymentIntent.allowRedisplay(PaymentSelection.CustomerRequestedSave.NoRequest)
-        ).isEqualTo(PaymentMethod.AllowRedisplay.UNSPECIFIED)
+        testAllowRedisplayValueForCustomerRequestedSave(
```

Comment:
In the future, could you do refactors like this as a separate PR before this one?